core-services/prow/02_config/securesign/fulcio/_prowconfig.yaml

```diff
@@ -0,0 +1,15 @@
+tide:
+  queries:
+    labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - jira/invalid-bug
+    - needs-rebase
+    repos:
+    - securesign/fulcio
+  
```

Comment:
@tommyd450 this is causing `ci/prow/yamllint` and `ci/prow/prow-config` to fail. Please remove the additional whitespace on line 15, but keep the newline.